src/main/java/org/myrobotlab/framework/repo/Repo.java

```diff
@@ -290,7 +290,7 @@ public String getFullTypeName(String serviceType) {
       }
      
 
-      ServiceType st = sd.getServiceType(type);
+      MetaData st = sd.getServiceType(type);

```

Comment:
the method name for getServiceType should probably be renamed?  It doesn't return a ServiceType anymore.

Reply:
Agree, updated